### PR TITLE
LPD-41022

### PR DESCRIPTION
**Pull request by brianikim**
https://liferay.atlassian.net/browse/LPD-41022,

@smottal , tied to LPP

### CODE REVIEW

**Comment by brianikim** [comment]
ci:test:sf

**Comment by brianikim** [comment]
ci:test:sf

**Comment by brianikim** [comment]
ci:test:playwright

**Comment by brianikim** [comment]
ci:stop

**Comment by brianikim** [comment]
ci:test:sf

**Comment by brianikim** [comment]
ci:test:playwright

File: modules/test/playwright/tests/users-admin-web/usersAndOrganizations.spec.ts [hunk 1]
```diff
@@ -474,3 +474,17 @@ test('LPD-29981 Check custom field is escaped', async ({
 
 	expect(customFieldDropDownOptions).toBeTruthy();
 });
+
+test('LPD-41022 Check toolbar select is disabled', async ({
+	usersAndOrganizationsPage,
+}) => {
+	await usersAndOrganizationsPage.goToUsers();
+	await expect(
```
Comment (smottal) [inline modules/test/playwright/tests/users-admin-web/usersAndOrganizations.spec.ts:482]: please add a new line before and after this

File: modules/test/playwright/tests/users-admin-web/usersAndOrganizations.spec.ts [hunk 2]
```diff
@@ -474,3 +474,17 @@ test('LPD-29981 Check custom field is escaped', async ({
 
 	expect(customFieldDropDownOptions).toBeTruthy();
 });
+
+test('LPD-41022 Check toolbar select is disabled', async ({
```
Comment (smottal) [inline modules/test/playwright/tests/users-admin-web/usersAndOrganizations.spec.ts:478]: is not disabled, is not visible at all

**Review by smottal — CHANGES_REQUESTED** [review]
@brianikim just very little changes

**Comment by smottal** [comment]
@brianikim after fixing these feel free to run the CI

**Comment by brianikim** [comment]
ci:test:sf

**Comment by brianikim** [comment]
ci:test:relevant

**Comment by brianikim** [comment]
ci:test:playwright

**Comment by smottal** [comment]
ci:forward

**Pull request closed without merging**